sklearn/gaussian_process/_gpc.py

```diff
@@ -496,6 +496,14 @@ class GaussianProcessClassifier(ClassifierMixin, BaseEstimator):
     classifiers are fitted. Note that this class thus does not implement
     a true multi-class Laplace approximation.
 
+    For an example on how to compare classification probability with
```

Comment:
For consistency with other PRs could you move the reference to the end of the "Examples" section of the docstrings? Also, could you shorten the description to something like, "For a comparison of the GaussianProcessClassifier with other classifiers, see ..."

The same applies to the docstrings of the other classifiers accordingly.

Reply:
That is now complete for all classes. Thank you :)